charts/linkerd2/values.yaml

```diff
@@ -97,6 +97,9 @@ proxy:
   # hooks](https://kubernetes.io/docs/concepts/containers/container-lifecycle-hooks/#container-hooks)
   # for more info on container lifecycle hooks.
   waitBeforeExitSeconds: 0
+  #  -- If set, the application container will not start until the proxy is
+  # ready
+  await: true
```

Comment:
This enables the feature by default, but allows users to disable with `--set proxy.await='false'`.

Reply:
@kleimkuhler How can we disable it with the CLI install method?

Reply:
`linkerd install --set proxy.wait=false`